src/imagery/i.eodag/i.eodag.html

```diff
@@ -52,7 +52,8 @@ <h4>Searching using scenes IDs</h4>
 Users have a set of scenes IDs that they want to search for and download.
 They can either use the <b>id</b> option or use the <b>file</b> option
 and pass a text file, with one ID per line.
-In both cases, specifying a provider is optional. In case users do
+When searching with ID, user must specify the product type that the IDs belong to.
+Specifying a provider is optional. In case users do
```

Comment:
```suggestion
When searching with ID, users must specify the product type that the ID 
belongs to. Specifying a provider is not mandatory, but it is recommended. In case users do
```